internals/cli/cli.go

```diff
@@ -300,7 +302,10 @@ func Run(options *RunOptions) error {
 		return fmt.Errorf("cannot create client: %v", err)
 	}
 
-	parser := Parser(cli)
+	if options.PebbleDir == "" {
+		options.PebbleDir, _ = getEnvPaths()
```

Comment:
Good point! Also, it's wrong currently (because the default `Logger` and `ClientConfig` aren't applied).

Probably needs to be something like this:

```golang
        pebbleDir := options.PebbleDir
        if pebbleDir == "" {
                pebbleDir, _ = getEnvPaths()
        }
        parser := Parser(cli, &RunOptions{
                ClientConfig: config,
                Logger: log,
                PebbleDir: pebbleDir,
        })
```